io.catenax.part_as_planned/2.0.0/PartAsPlanned.ttl

```diff
@@ -0,0 +1,141 @@
+   samm:dataType :PartSitesInformationEntity.
+
+:PartTypeInformationEntity a samm:Entity;
+   samm:preferredName "Part Type Information Entity"@en;
+   samm:description "Encapsulation for data related to the part type"@en;
+   samm:properties(:manufacturerPartId :nameAtManufacturer :classification).
+
+:PartSitesInformationEntity a samm:Entity;
+   samm:preferredName "Part Sites Information Entity"@en;
+   samm:description "Describes the ID, function and validity date of a site for the associated part in the AsPlanned context."@en;
+   samm:properties(:catenaXsiteId :function [ samm:property :functionValidFrom; samm:optional true ] [ samm:property :functionValidUntil; samm:optional true ]).
+
+:manufacturerPartId a samm:Property;
+   samm:preferredName "Manufacturer Part ID"@en;
+   samm:description "Part ID as assigned by the manufacturer of the part. The Part ID identifies the part in the manufacturer`s dataspace. The Part ID references a specific version of a part. The version number must be included in the Part ID if it is available.\n\nThe Part ID does not reference a specific instance of a part and must not be confused with the serial number."@en;
+   samm:characteristic :PartIdCharacteristic;
+   samm:exampleValue "123-0.740-3434-A".
+
+:nameAtManufacturer a samm:Property;
+   samm:preferredName "Name at manufacturer"@en;
+   samm:description "Name of the part as assigned by the manufacturer"@en;
+   samm:characteristic :PartNameCharacteristic;
+   samm:exampleValue "Mirror left".
+
+:classification a samm:Property;
+   samm:preferredName "Classifcation"@en;
+   samm:description "The classification of the part type according to STEP standard definition"@en;
+   samm:characteristic :ClassificationEnumerationCharacteristic;
+   samm:exampleValue "software".
+
+:catenaXsiteId a samm:Property;
+   samm:preferredName "Catena-X site identifier"@en;
+   samm:description "The identifier of the site according to Catena-X BPDM. The catenaXsiteId must be a valid Catena-X BPN. The BPN is a unique, unchangeable identifier for Business Partners / company locations from foundation to closure, regardless of the different business relationships / structures between or within the Business Partners or company locations."@en;
+   samm:characteristic ext-number:BpnsCharacteristic;
+   samm:exampleValue "BPNS1234567890ZZ".
+
+:function a samm:Property;
+   samm:preferredName "Function"@en;
+   samm:description "The function of the site in relation to the part(i.e. the activity within the value chain of the part that is performed at the site)"@en;
+   samm:characteristic :FunctionCharacteristic;
+   samm:exampleValue "production".
+
+:functionValidFrom a samm:Property;
+   samm:preferredName "Function valid from"@en;
+   samm:description "Timestamp, from when the site has the specified function for the given part"@en;
+   samm:characteristic samm-c:Timestamp.
+
+:functionValidUntil a samm:Property;
+   samm:preferredName "functionValidUntil"@en;
```

Comment:
preferredName should not be in camel notation. just make it simple English word.